docs/features/telemetry-serilog.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Home"
+layout: default
+---
+
+# Telemetry (Serilog)
```

Comment:
Let's maybe dedicate this page to all enrichers only? We'll have a few over time.